TTHAnalysis/python/plotter/susy-sos/mcc_sos.txt

```diff
@@ -10,6 +10,7 @@ LepGood1_(\w+)		: LepGood_\1[iLepFO_Recl[0]]; AlsoData
 LepGood2_(\w+)		: LepGood_\1[iLepFO_Recl[1]]; AlsoData
 LepGood3_(\w+)		: LepGood_\1[iLepFO_Recl[2]]; AlsoData
 LepGood4_(\w+)		: LepGood_\1[iLepFO_Recl[3]]; AlsoData
+JetSel1_(\w+)		: JetSel_\1[iJetSel_Recl[0]]; AlsoData
```

Comment:
For the sake of simplicity, maybe we could drop write `Jet1_(\w+)` instead of `JetSel1_(\w+)`, which is also in accordance with line 16 of this file?